jylibs/commands.py

```diff
@@ -410,36 +406,16 @@ def proxygen(sArgs=None, rArgs=None):
 		name = "mailboxd",
 		cmd  = exe["MAILBOXD"] + " %s",
 	),
-	"zimbra" : Command(
-		desc = "zimbra",
-		name = "zimbra",
-		cmd  = exe["ZIMBRA"] + " %s",
-		),
-	"zimbraadmin" : Command(
-		desc = "zimbraadmin",
-		name = "zimbraadmin",
-		cmd  = exe["ZIMBRAADMIN"] + " %s",
-	),
 	"service" : Command(
 		desc = "service",
 		name = "service",
 		cmd  = exe["SERVICE"] + " %s",
 	),
-	"zimlet" : Command(
-		desc = "zimlet",
-		name = "zimlet",
-		cmd  = exe["ZIMLET"] + " %s",
-	),
 	"ldap" : Command(
 		desc = "ldap",
 		name = "ldap",
 		cmd  = exe["LDAP"] + " %s",
 	),
-	"logger" : Command(
```

Comment:
Was the logger completely removed?

Reply:
Yes, the logger was removed in the past but they forgot to clean this up.